src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -456,23 +456,40 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
 
         List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
 
+        String instructorFragmentArgumentInstructors = "<p>The email below has been sent to students of course: "
+                + SanitizationHelper.sanitizeForHtml(course.getId()) + ", "
+                + SanitizationHelper.sanitizeForHtml(course.getName())
+                + ".<br>" + Const.EOL + "<br>" + Const.EOL
+                + "=== Email message as seen by the students ===</p>" + Const.EOL;
```

Comment:
@leeyimin I do prefer the first way, so I will do it that way (by extracting a method) and I will let you know, ok?